### PR TITLE
Mwpw 137051 breadcrumb cls

### PR DESCRIPTION
**Pull request by vhargrave**
Based on this PR - https://github.com/adobecom/express/pull/498
Reserves some extra spacing for pages with breadcrumbs to lower CLS on those pages

Resolves: [MWPW-137051](https://jira.corp.adobe.com/browse/MWPW-137051)

Test URLs:
- Before: https://main--express--adobecom.hlx.page/express/?martech=off
- After: https://MWPW-137051-breadcrumb-cls--express--adobecom.hlx.page/express/?martech=off

### CODE REVIEW

**Review by apganapa — APPROVED** [review]

File: express/scripts/gnav.js [hunk 1]
```diff
@@ -108,45 +148,7 @@ async function loadFEDS() {
...
-    const breadCrumbList = [secondBreadCrumb];
-
-    if (pathSegments.length >= 3) {
-      const thirdBreadCrumb = buildBreadCrumb(pagesShortName, pagesShortName, secondBreadCrumb.url);
-      breadCrumbList.push(thirdBreadCrumb);
-    }
-    return breadCrumbList;
-  }
-
+  // eslint-disable-next-line import/prefer-default-export
```
Comment (qiyundai) [inline express/scripts/gnav.js:151]: Then we probably don't need this anymore 😄

**Review by qiyundai — COMMENTED** [review]
Looks pretty good. Is this PR still waiting to be merged?

File: express/scripts/gnav.js (hunk 1 above)
Comment (vhargrave) [reply express/scripts/gnav.js:151]: yes it is! :D

Comment (qiyundai) [reply express/scripts/gnav.js:151]: I can push a commit to remove this is you don't mind 😄

**Review by qiyundai — APPROVED** [review]

File: express/scripts/gnav.js (hunk 1 above)
Comment (vhargrave) [reply express/scripts/gnav.js]: thanks :D

**Pull request merged**